InventoryIndexer/Indexer/IndexHandler.php

```diff
@@ -66,7 +67,7 @@ public function saveIndex(IndexName $indexName, \Traversable $documents, string
 
         $columns = [IndexStructure::SKU, IndexStructure::QUANTITY, IndexStructure::IS_SALABLE];
         foreach ($this->batch->getItems($documents, $this->batchSize) as $batchDocuments) {
-            $connection->insertArray($tableName, $columns, $batchDocuments);
+            $connection->insertArray($tableName, $columns, $batchDocuments, AdapterInterface::INSERT_IGNORE);
```

Comment:
The 4th parameter is missing in the `\Magento\Framework\DB\Adapter\AdapterInterface::insertArray` method signature, it means that such implementation will be broken in case when someone replaces implementation of `\Magento\Framework\DB\Adapter\AdapterInterface`.

Reply:
I think that replacement implementation of \Magento\Framework\DB\Adapter\AdapterInterface::insertArray in \Magento\Framework\DB\Adapter\Pdo\Mysql::insertArray will be very expensive, because it is Magento library and it is used in many places in core, and similar case with 4th parameter such as my fix is implemented in core many times. So maybe you can offer a better solution?

Reply:
I changed my fix, you can check it.